release-notes/next-release.md

```diff
@@ -22,3 +22,5 @@ of SBO:0000633 (see https://sourceforge.net/p/sbo/term-request/113/)
 * Removed `model.add_reaction()` and replaced remaining usages of it with `model.add_reactions()`
 * Removed the following tests: test_add_remove_reaction_benchmark, test_add_reaction, test_add_reaction_context, test_add_reaction_from_other_model, test_add_cobra_reaction
 * Removed `model.__add__` and `model.__iadd__` - use `model.merge` to replace them.
+* Remove `Model().description()`.
```

Comment:
Was this done in this PR? I don't see a code change to that effect.

Reply:
Oh I see, that's from the other PR right?

Reply:
Right, this one was undocumented.